spec/functional/rebooter_spec.rb

```diff
@@ -43,7 +43,7 @@ def create_resource
 
   let(:expected) do
     {
-      :windows => 'shutdown /r /t 300 /c "rebooter spec test"',
+      :windows => "#{ENV['SystemDrive']}/Windows/System32/shutdown.exe /r /t 300 /c \"rebooter spec test\"",
```

Comment:
I guess the test that uses this will weirdly work on non-windows platforms because the variable will be empty in both cases, so, maybe that's okay, but I'd really expect the test below to be filtered using `:windows_only`